js/components/ProfileQuickView.jsx

```diff
@@ -75,18 +64,17 @@ var ProfileQuickView = React.createClass({
     var imgUrl = strings.DEFAULT_USER_PIC;
     var expertCategories = '';
     var investorCategories = '';
-    if (this.state.currentUser) {
```

Comment:
I think what you should do is keep this if statement but change state to props, but I'm not positive.